### DISCUSSION TITLE
HowTo include only some folders

### DISCUSSION THREAD

**Discussion by KamikazeePL** [filebrowser/filebrowser#2657] [Show and tell]
First exclude everything with
`/.+` and 
 - uncheck **Allow** 
 + check **Regex**

then add folders to share (to be visible)
`/addThisFolder`
 + check **Allow**
 - uncheck **Regex**
![filebrowser-exclude](https://github.com/filebrowser/filebrowser/assets/35179986/5050e2f7-615f-4cd7-be60-270c6ceb4d9f)

**Comment by Marshin386** [comment]
This is a handy way to give access to only select folders, however there is an issue with it. You can only set the allow folders to top level folders, if you try to set it to a seb folder, using your example /downloads/programs , it won't work. It effectively ignores it. If I'm missing something to make this work please let me know, if this is a bug I'll update as soon as it's fixed. This is much needed.
